### DISCUSSION TITLE
Time counter not reset on track change

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#57] [Fixed Issues]
**rAudio 1 • 20201231**

In the playback window the time counter won't reset to 0:00 on track change. Instead the counter remains at the end value of the previous track. The issue is valid both in progress bar and time circle mode.

**Comment by klassikmann** [comment]
**rAudio 1 • 20210101** contains an improvement and a regression.
Improvement: The Problem is fixed, but only in progress bar mode.
Regression: On small devices setting time circle mode has no effect.

**Reply by R101G** [reply]
I am playing FLAC files. The progress bar disappears after the first track, and there is no elapsed/track-time shown. Unticking the progress bar in setup has no effect - it's still there for the first track.

Edit: If I go to Library or Playlist pages and return to the playback page, the time info appears. Refreshing the page hides it again.

Also on the Playback page, the bit-depth and sample rate display is useful, but kbit/s appears to be a carry-over from lossy compression sources.

**Reply by klassikmann** [reply]
> **rAudio 1 • 20210101** contains an improvement and a regression.
> Improvement: The Problem is fixed, but only in progress bar mode.

**rAudio 1 • 20210102** fixes the issue.

> Regression: On small devices setting time circle mode has no effect.

Using the small device in landscape mode the time circle is displayed, but not in portrait mode, where the progress bar is presented. I could live with that because the progress bar is convenient, but I need an info (@rern please comment) whether this effect is intended.

**Reply by klassikmann** [reply]
**rAudio 1 • 202101021** does not change the behaviour regarding "Regression".

**Reply by R101G** [reply]
I can confirm what I think is a related problem. I do not have volume control or time enabled on the playback page. On my 7" tablet, in portrait mode I get both top and bottom bar and the mode and track time line displayed. In landscape, I lose the mode/track time line unless I hide the top/bottom bars. Same for the radio symbol when using webradio.

When in system settings, I have only the top half  of the the switch-page icons shown at the bottom of the screen in portrait mode.

**Comment by rern** [comment]
Time-knob is hidden in portrait mode with width less than 614px.

**Reply by klassikmann** [reply]
> Time-knob is hidden in portrait mode with width less than 614px.

@rern Hm, the screen resolution of my smartphone is 1440x2560.

**Comment by rern** [comment]
Browsers see it as (1440/4) x (2560/4) > 360x640.

**Reply by klassikmann** [reply]
> Browsers see it as (1440/4) x (2560/4) > 360x640.

@rern Interesting. Can you explain in some more detail? My tablet has a screen resolution of 2048x1536 without a hidden time-knob.

EDIT: The volume-knob is not disappearing, only the time-knob, again only on the smartphone, not on the tablet.

EDIT2: Question: How does the size of the folder.jpg affect the position or hiding of the knobs?

**Reply by rern** [reply]
Try capturing a desktop screen and resize to the size of your phone to see why the resolutions cannot be as of physical.
Drag resizing browser to see the responsive layout in real time.

**Reply by klassikmann** [reply]
@rern I'm assuming your implementation is correct, but I don't fully understand it yet. Here are three screenshots of my 1440x2560 smartphone scaled down by a factor 4. Both time- and volume-knob are configured.
1. landscape
![landscape](https://user-images.githubusercontent.com/25383513/103478545-c8f0ff80-4dc7-11eb-8eee-49c15ef4cc51.jpg)
Both knobs are present in landscape mode
2. portrait top  
![portrait-top](https://user-images.githubusercontent.com/25383513/103478566-e8882800-4dc7-11eb-9001-0f310c0a3de6.jpg)
3. portrait bottom
![portrait-bottom](https://user-images.githubusercontent.com/25383513/103478581-f8a00780-4dc7-11eb-870c-7608130abac7.jpg)
Only the volume-knob is present in potrait mode

I have to concede that I'm using the time-knob for testing purposes only. Hence I can live with the state of the implementation, be it a bug or a unavoidable contraint.

**Reply by rern** [reply]
Your screenshots show correct layouts.
The progress bar can be tapped or dragged. Tell me what you cannot do without the time knob.

**Reply by klassikmann** [reply]
> Your screenshots show correct layouts.
> The progress bar can be tapped or dragged. Tell me what you cannot do without the time knob.

@rern As I wrote "_I have to concede that I'm using the time-knob for testing purposes only._" and preferring the progress bar on the smartphone I'll close the bug report (i.e. changing the category).
